lib/tokenize.cpp

```diff
@@ -2707,7 +2707,7 @@ void Tokenizer::setVarIdPass1()
             if (!isC() && Token::simpleMatch(tok2, "const new"))
                 continue;
 
-            bool decl;
+            bool decl = false;
```

Comment:
This is a false positive from VC++. The compiler fails to see that `syntaxError()` function never returns.

Reply:
removed commit with this change